Makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: all
+all: clean build-bin build-docker deploy
+
+.PHONY: help
+help:
+	@echo
+	@echo "\033[92m  Usage:"
+	@printf "    \033[36m%-15s\033[93m %s" "make" "<target>"
+	@echo
+	@echo
+	@echo "\033[92m  Targets: "
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "    \033[36m%-15s\033[93m %s\n", $$1, $$2}'
+	@printf "\033[0m"
+
+.PHONY: clean 
+clean:
+	go cleana
```

Comment:
I think there is a typo here? `go clean`

Reply:
Yep thanks.